packages/jest-util/src/buffered_console.js

```diff
@@ -41,6 +41,7 @@ export default class BufferedConsole extends Console {
     message: LogMessage,
     level: ?number,
   ) {
+    // TODO: This callsite needs to read sourcemaps
```

Comment:
The issue is here: https://github.com/facebook/jest/blob/430aebe49e9f144d29f144f5a29482315e269e4a/packages/jest-runner/src/run_test.js#L86-L108

I need to pass in `runtime` to the console, but `environment` requires `testConsole` and the `runtime` requires `environment`